<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ddtt_widget</class>
 <widget class="QWidget" name="ddtt_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>168</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="4" column="1">
       <widget class="QPushButton" name="testButton">
        <property name="text">
         <string>Execute...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="metricBox">
        <item>
         <property name="text">
          <string>ALL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>POSITIONAL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HEIGHT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ROTATIONAL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SCALING</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>STRUCTURE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DISTORTION</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Metric</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Node</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="nodeIndex"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Splits</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="splits">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>512</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="bestAssign">
        <property name="text">
         <string>Show best assignment</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QPushButton" name="correspondButton">
        <property name="text">
         <string>Correspond..</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QPushButton" name="loadGraphs">
        <property name="text">
         <string>Load..</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="experimentButton">
        <property name="text">
         <string>Experiment..</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
